psydac/api/fem.py

```diff
@@ -187,24 +187,38 @@ def __init__(self, expr, kernel_expr, *args, **kwargs):
         # ...
         if len(domain)>1:
             i,j = self.get_space_indices_from_target(domain, target )
-            test_space  = self.spaces[0].spaces[i]
-            trial_space = self.spaces[1].spaces[j]
+            trial_space  = self.spaces[0].spaces[j]
+            test_space = self.spaces[1].spaces[i]
```

Comment:
Is this a bug fix?

Reply:
yes

Reply:
Good! Please add a word about this in the PR description

Reply:
done